app/lib/tariff_synchronizer.rb

```diff
@@ -324,5 +328,13 @@ def check_tariff_updates_failures
                  file_names: BaseUpdate.failed.map(&:filename))
       raise FailedUpdatesError
     end
+  rescue FailedUpdatesError => e
+    notify_slack_app(e)
+
+    raise
+  end
+
+  def notify_slack_app(exception)
```

Comment:
Should we add a test for this integration with the service? I suspect this isn't actually passing?

Reply:
Just a light touch one:

allow(slack_notifier_service).to receive(:call)
expect(slack_notifier_service).to have_receive(:call).with(expected_message)